CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - /deliveryservice_user
   - /deliveryservice_user/:dsId/:userId
   - /deliveryservices/{dsid}/regexes/{regexid} (GET)
+  - /deliveryservices/:id
```

Comment:
nit but can you add (GET) to the end of this since it still supports PUT and DELETE?